vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/post_credentials/post_credentials.cc

```diff
@@ -82,13 +82,13 @@ void PostCredentials::Request(
       this,
       _1,
       callback);
-  ledger_->LoadURL(
-      GetUrl(order_id),
-      {},
-      GeneratePayload(item_id, type, std::move(blinded_creds)),
-      "application/json; charset=utf-8",
-      ledger::UrlMethod::POST,
-      url_callback);
+
+  auto request = ledger::UrlRequest::New();
+  request->url = GetUrl(order_id);
+  request->content = GeneratePayload(item_id, type, std::move(blinded_creds));
+  request->content_type = "application/json; charset=utf-8";
```

Comment:
Maybe this could be a default too if it's unchanged 99% of the time? Or at a minimum, it could be a mojom definition so that we don't accidentally introduce typos in a bad copy/paste.

Reply:
default is empty as most of requests are get